demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -295,6 +301,16 @@ def run_doc_review(self):
 
         for file in self.files:
             click.echo(f"\nChecking file {file}")
+
+            # --xsoar-only flag is specified.
+            # If the file is a path
+            if self.is_xsoar_supported_rn_only and not is_xsoar_supported_pack(file):
+                click.secho(
+                    f"File '{file}' was skipped because it does not to belong to an XSOAR-supported Pack",
+                    fg="yellow",
+                )
+                continue
```

Comment:
Not a fan of `continue` statements, but the decision is yours.

Reply:
Interesting. Why is that? Because it makes the code harder to follow? 

I agree that in theory, when we have nested loops and functions inside a method, it would make the code harder to predict.

However, I think in this case, since the function `run_doc_review` has only one real iteration (the one where we loop over all files that need to be checked for spelling), introducing another variable to hold the result and move on to the next file to be checked would be an overkill (if this is what you're suggesting).

Reply:
Generally, continue statements are considered bad programming practice since they effect the run of the code outside their scope. There is some discussion of this here if you are interested.

https://softwareengineering.stackexchange.com/questions/58237/are-break-and-continue-bad-programming-practices

I don't think it would be an overkill to hold another variable but I don't mind if you decide to leave it like that.